eng/scripts/splittestdependencies/tests/expectOutputs/expect-matrix-integration.json

```diff
@@ -15,14 +15,14 @@
         "Pool": "azsdk-pool-mms-ubuntu-2004-general",
         "TestTargetFramework": "net6.0"
       },
-      "Windows2019_NET6.0": {
+      "windows2022_NET6.0": {
```

Comment:
@sima-zhu Are these tests still passing or is there a hardcoded input/output/expect for 2019 in the test file itself?

Reply:
Let me find the test pipeline related to this change

Reply:
Fixed failed tests already